recipes/leftbar/general_row/general_row.vue

```diff
@@ -12,11 +12,13 @@
       <div
         class="dt-leftbar-row__alpha"
       >
-        <dt-recipe-leftbar-general-row-icon
-          :type="getIcon"
-          :color="color"
-          data-qa="dt-leftbar-row-icon"
-        />
+        <slot name="left">
+          <dt-recipe-leftbar-general-row-icon
+            :type="getIcon"
+            :color="color"
+            data-qa="dt-leftbar-row-icon"
+          />
+        </slot>
```

Comment:
Slot is pretty much what I thought of this as from General, and supports everything else about other Leftbar Recipes. Defer to @braddialpad.

Reply:
I'm not sure I understand the question here, but I think this is a good pattern.

Reply:
My comment was just me saying "cool cool cool". No question.